docs/user/reference/metrics/timing_distribution.md

```diff
@@ -524,7 +567,16 @@ assert_eq!(
 ```
 
 </div>
-<div data-lang="JavaScript" class="tab" data-bug="1716954"></div>
+<div data-lang="JavaScript" class="tab">
+
+```Javascript
+import * as timing from "./path/to/generated/files/timing.js";
+import { ErrorType } from "@mozilla/glean/<platform>";
+
+assert.equal(1, await metric.testGetNumRecordedErrors(ErrorType.InvalidValue));
```

Comment:
While I can make the leap to see where `metric` comes from here, perhaps calling it as a member of the `timing` module would be clearer (or even using the same example metric as used in the `testGetValue` example):

```suggestion
assert.equal(1, await timing.metric.testGetNumRecordedErrors(ErrorType.InvalidValue));
```

Reply:
Should be fixed in https://github.com/mozilla/glean/commit/f240b4a8055146e69b5360e8544297af9b0dc5df